src/apme_engine/remediation/graph_engine.py

```
@@ -40,6 +40,7 @@
 
 ProgressCallback = Callable[[str, str, float, int], None]
 RescanFn = Callable[[ContentGraph, frozenset[str]], Awaitable[list["ViolationDict"]]]
+AiPhaseStartCallback = Callable[[list[ViolationDict]], None | Awaitable[None]]
```

Ruff (0.16.0)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)